Templates/AppSource App/.github/workflows/NextMajor.yaml

```diff
@@ -0,0 +1,157 @@
+name: Test Next Major
+
+on:
+  workflow_dispatch:
+
+defaults:
+  run:
+    shell: PowerShell
+
+jobs:
+  Initialization:
+    runs-on: [ windows-latest ]
+    outputs:
+      telemetryScopeJson: ${{ steps.init.outputs.telemetryScopeJson }}
+    steps:
+      - id: init
+        uses: microsoft/AL-Go-Actions/WorkflowInitialize@main
+        with:
+          eventId: "DO0099"
+
+  Analyze:
+    runs-on: [ windows-latest ]
+    needs: [ Initialization ]
+    outputs:
+      projects: ${{ steps.Projects.outputs.projects }}
+      projectCount: ${{ steps.Projects.outputs.projectCount }}
+      githubRunner: ${{ steps.OutputSettings.outputs.githubRunner }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Read settings
+        uses: microsoft/AL-Go-Actions/ReadSettings@main
+        with:
+          get: githubRunner,alwaysBuildAllProjects
+
+      - name: Output settings
+        id: OutputSettings
+        run: |
+          $gitHubRunner = "$env:githubRunner".Split(',') | ConvertTo-Json -compress
```

Comment:
looks like some code is duplicated from the other yaml file. can it be refactored ?